tendrl/node_agent/node_sync/__init__.py

```diff
@@ -62,6 +62,28 @@ def _run(self):
                 tendrl_ns.node_agent.objects.NodeContext(tags=tags).save()
                 gevent.sleep(interval)
 
+                if tendrl_ns.tendrl_context.integration_id:
+                    try:
+                        tendrl_ns.etcd_orm.client.read(
+                            "/clusters/%s" % (
+                                tendrl_ns.tendrl_context.integration_id
+                            )
+                        )
+                    except etcd.EtcdKeyNotFound:
+                        pass
```

Comment:
Need a log.error here saying that local tendrl context not found in central store